pipeline/src/pipeline-config/outboxEventPublisher.ts

```diff
@@ -41,11 +41,18 @@ export async function publishError
 }
 
 export async function publishSuccess
-(client: ClientBase, pipelineId: number, pipelineName: string, result: unknown): Promise<string> {
-  const content = {
+(client: ClientBase, pipelineId: number, pipelineName: string, result: unknown, schema?: object): Promise<string> {
+  let content: any = {
     pipelineId: pipelineId,
     pipelineName: pipelineName,
     data: result
   }
+  if (schema !== undefined || schema !== null) {
+    content = {
+      ...content,
+      schema: schema as object
```

Comment:
You could make it shorter by
```
const content = {
    pipelineId: pipelineId,
    pipelineName: pipelineName,
    data: result,
    schema: schema != null ? schema as object : undefined
}
```

or just do
```
content.schema = schema as obejct;
``` 
and provide a schema for `content` (pls don't use any)